plenum/test/node_request/test_commit/test_num_of_commit_with_f_plus_one_faults.py

```diff
@@ -38,8 +38,9 @@ def afterElection(setup, up):
 def testNumOfCommitMsgsWithFPlusOneFaults(afterElection, looper,
                                           nodeSet, prepared1, noRetryReq):
     with pytest.raises(AssertionError):
+        # To raise an error pass less than the actual number of faults
         checkCommitted(looper,
                        nodeSet,
                        prepared1,
                        range(getNoInstances(len(nodeSet))),
-                       faultyNodes)
+                       faultyNodes-1)
```

Comment:
Because master uses a branch of stp, that has an incorrect implementation of eventuallyAll and a bug in eventually which makes the test work beyond given timeouts, also `eventuallyAll` does not give guarantee that functions will run. if you look at the underlying logic of `checkCommitted` it does not call `eventuallyAll` with value of `faultyNodes` so the first failure gives the error but `chk_all_funcs` takes into account `faultyNodes`. What this test is checking is the behavior of `msgCountOK` if given wrong parameters which is silly